nvim/after/ftplugin/tex.lua

```diff
@@ -1,23 +1,19 @@
-local M = {}
```

Comment:
This `local M = {}` and `return M` is used to export functions within a module. I needed this to export my plugin's functionality to the end user, whereas you only need this if you want a file's functions to be accessible from another file.